src/lib/containers/TermsAndConditions.tsx

```diff
@@ -145,8 +148,8 @@ const TermsAndConditions: React.FunctionComponent<TermsAndConditionsProps> = ({
       <h3 className="page-header">Synapse Pledge</h3>
       <form>
         <label>
-          I affirm my commitment to all Synapse Governance policies for
-          responsible research and data handling (linked below), including:
+          I affirm my commitment to all <Link>Synapse Governance policies</Link>{' '}
```

Comment:
The link to Sage Governance Policies does not exist but will be a Confluence page

Reply:
Yes, I thinks the link provided would work

Reply:
Could you verify with Adam that this link is ok (who should verify this with the Governance team)?

Reply:
Confirmed the link with Adam